<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QWidget" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BLE Communication</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QGridLayout" name="leftLayout">
     <item row="11" column="0">
      <widget class="QPushButton" name="connector_bottom_left">
       <property name="text">
        <string>BOTTOM LEFT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="12" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>BOTTOM</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="connector_top_left">
       <property name="text">
        <string>TOP LEFT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="connector_top_right">
       <property name="text">
        <string>TOP RIGHT</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="connector_bottom_right">
       <property name="text">
        <string>BOTTOM RIGHT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="connector_top">
       <property name="text">
        <string>TOP</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2" alignment="Qt::AlignVCenter">
      <widget class="QSlider" name="slider1">
       <property name="value">
        <number>49</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="slider2">
       <property name="value">
        <number>49</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="slider0">
       <property name="value">
        <number>49</number>
       </property>
       <property name="sliderPosition">
        <number>49</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="sliderText0"/>
     </item>
     <item row="6" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QLineEdit" name="input">
       <property name="text">
        <string>24:7F:0E:E6:AE:34</string>
       </property>
       <property name="placeholderText">
        <string>Enter IP Address...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_button">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnect_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start Communication</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Communication</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message_input">
       <property name="placeholderText">
        <string>Enter Message...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="output">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
